spec/models/collection_spec.rb

```diff
@@ -315,6 +381,45 @@
       end
     end
 
+    context "when the collection contains a public bookmark rejected by collection" do
+      let(:bookmark) { create(:bookmark) }
+
+      before do
+        bookmark.collections << collection
+        bookmark.collection_items.update!(collection_approval_status: :rejected)
+      end
+
+      it "does not count the bookmark" do
+        expect(collection.general_bookmarked_items_count).to eq(0)
+      end
+    end
+
+    context "when the collection contains a public bookmark rejected by user" do
+      let(:bookmark) { create(:bookmark) }
+
+      before do
+        bookmark.collections << collection
+        bookmark.collection_items.update!(user_approval_status: :rejected)
+      end
+
+      it "does not count the bookmark" do
+        expect(collection.general_bookmarked_items_count).to eq(0)
+      end
+    end
+
+    context "when the collection contains an invited public bookmark" do
```

Comment:
```suggestion
    context "when the collection contains a public bookmark awaiting mod approval" do
```
Or some other, better phrasing that reflects the changes to the bookmark tests. 

Once these are updated, anyone can hit the big green merge button. I've got to go to sleep.